### DISCUSSION TITLE
Simple multilingual website module with handlebars

### DISCUSSION THREAD

**Discussion by sihamlamssaoui** [npm/feedback#297] [Website] [answered]
Hi how can set multilingual with handlebars please

**Comment by MylesBorins** [comment, marked as answer]
Hey @sihamlamssaoui . This discussions form is for suggesting features for the npm registry, website, and CLI. We do not offer general technical support through this channel. You may want to try opening an issue over at https://github.community or on Stackoverflow. Best of luck!
